internal/core/monitoring/metrics.go

```diff
@@ -73,7 +76,7 @@ func CreateGaugeMetric(options *MetricOpts) *prometheus.GaugeVec {
 		prometheus.GaugeOpts{
 			Namespace: options.Namespace,
 			Subsystem: options.Subsystem,
-			Name:      options.Name + "_gauge",
+			Name:      options.Name + "_" + options.MetricUnit + "_total",
```

Comment:
Hey @arthur29, you are following [prometheus naming conventions](https://prometheus.io/docs/practices/naming/) right? if so, it states that "an accumulating count has total as a suffix, in addition to the unit if applicable". Since gauge metric is not cumulative, this piece of code "unit + total" should be in counter metric only.

Reply:
Yeah I think that I understand bad the idea, and the unit should be on metrics that change its number (aka gauge)